### ISSUE TITLE
Bter making orders

### ISSUE THREAD

**Issue by marinsokol5** [kroitor/ccxt#115]
After some debugging I have finally found what is wrong with Bter's create_limit_order methods.

The symbol given isn't transformed into the product(now called market) used at Bter.
So calling 
` bter.create_limit_buy_order('ETH/BTC', 1, 0.0005) ` 
throws:
```
  response = getattr(self, method)(self.extend(order, params))
  File "/home/marin/fino/.venv/lib/python3.6/site-packages/ccxt/__init__.py", line 5501, in request
    raise ExchangeError(self.id + ' ' + self.json(response))
ccxt.ExchangeError: bter {"result":"false","code":6,"message":"Error: invalid data  "}
``` 
Which doesn't really tell us anything but calling 
`bter.create_limit_buy_order('eth_btc', 1, 0.0005)`
results in : 
`{'info': {'result': 'true', 'message': 'Success', 'code': 0, 'orderNumber': ****}, 'id': ***}`

If in method symbol instead of :
```
def symbol(self, market):
    p = self.market(market)
    return p['symbol'] if type(p) is dict else market
```
it returns `p['id']`  the order would pass just fine.

If the method symbol works on every other exchange then you could override it for Bter.

Cheers.

**Comment by kroitor** [comment]
Thanks very much for this report and for the solution! Will upload the fix for it very soon!

**Comment by kroitor** [comment]
```JavaScript
    async createOrder (market, type, side, amount, price = undefined, params = {}) {
        if (type == 'market') // ← this is also needed
            throw new ExchangeError (this.id + ' allows limit orders only');
        await this.loadMarkets ();
        let method = 'privatePost' + this.capitalize (side);
        let order = {
            'currencyPair': this.marketId (market), // ← this is the proper way, id not symbol
            'rate': price,
            'amount': amount,
        };
        let response = await this[method] (this.extend (order, params));
        return {
            'info': response,
            'id': response['orderNumber'],
        };
    },
```

**Comment by marinsokol5** [comment]
Thank you

**Comment by kroitor** [comment]
@marinsokol5 , no, thanks to you, without your help, it would have taken an undefined time from me to fix it! You're the man! It's available in 1.3.117 now! Feel free to contact me if you have any other difficulties. Best wishes!
